### PR TITLE
{lib}[intel/2015a] Hoard 3.10 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
@stdweird: please review?

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a381a19a8359319b452d for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/h/Hoard/Hoard-3.10-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+name = 'Hoard'
+version = '3.10'
+
+homepage = 'http://www.hoard.org/'
+description = """Hoard is a fast, scalable, and memory-efficient memory allocator that can speed up your applications."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['https://github.com/emeryberger/Hoard/releases/download/%(version)s']
+sources = ['Hoard-%(version)s-source.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/Hoard/Hoard-3.10-intel-2015a.eb]: add empty newline

**Comment by wpoely86** [comment]
looks fine

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**